src/main/scala/org/ergoplatform/modifiers/history/Header.scala

```diff
@@ -151,7 +155,7 @@ object HeaderSerializer extends Serializer[Header] {
     val stateRoot = ADDigest @@ bytes.slice(97, 130)
     val timestamp = Longs.fromByteArray(bytes.slice(130, 138))
     val votes = bytes.slice(138, 143)
-    val nBits = RequiredDifficulty.parseBytes(bytes.slice(143, 147)).get
+    val nBits = RequiredDifficulty.parseBytes(bytes.slice(143, 147)).getOrElse(throw new IllegalArgumentException("cannot parse NBITS"))
```

Comment:
.getOrElse(throw new) is not any different from .get and breaks function signature "getOrElse[B >: A](default: => B): B"